server/api/impl/aws/provision_data.go

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package aws
+
+import (
+	"archive/zip"
+	"bytes"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+)
+
+const bundleStaticAssetsFolder = "static/"
+
+type ProvisionData struct {
```

Comment:
Is there anything here TerraForrm specific or is it an internal data model shared with the cloud team?

Reply:
Nothing terraform specific just an internal model

Reply:
Should we document it somewhere?